opencti-platform/opencti-front/src/private/components/Dashboard.jsx

```diff
@@ -908,25 +905,47 @@ const CustomDashboard = ({ dashboard, timeField }) => {
     </Security>
   );
 };
+
+const changeDashboardPref = graphql`
+  mutation DashboardPrefMutation($input: [EditInput]!) {
+    meEdit(input: $input) {
+      dashboard_id
+    }
+  }
+`;
+
 const Dashboard = () => {
   const classes = useStyles();
-  const [view, saveView] = useViewStorage('view-dashboard');
-  const { dashboard = 'default', timeField = 'technical' } = view;
-  const handleChangeTimeField = (event) => saveView({ dashboard, timeField: event.target.value });
-  const handleChangeDashboard = (event) => saveView({ dashboard: event.target.value, timeField });
+  const { me: { dashboard_id: dashboardId } } = useContext(UserContext);
```

Comment:
useAuth()

Reply:
would you like to give me a little bit of info on why it is preferable to use `useAuth()` here?

Reply:
That's a choice we made to hide the UserContext as we use it a lot of time in the app and add some security.